programs/stake/src/stake_state.rs

```diff
@@ -303,7 +324,18 @@ fn calculate_stake_points_and_credits(
         }
     }
 
-    (points, new_credits_observed, false)
+    StakePointsEarned {
+        points,
+        new_credits_observed,
+        force_credits_update_with_skipped_reward: false,
+    }
+}
+
+#[derive(Debug, PartialEq)]
+struct StakeRewardDistributions {
```

Comment:
bikeshed: how about `DistributedStakeRewards` or `CalculatedStakeRewards`?

Reply:
I went with the `Calculated` prefix for both structs